app/controllers/api/v1/forms_controller.rb

```diff
@@ -61,6 +61,13 @@ def make_unlive
     end
   end
 
+  def archive
+    form.archive_live_form!
+    render json: form, status: :ok
+  rescue AASM::InvalidTransition
+    render json: { error: "Form cannot be archived" }, status: :forbidden
+  end
```

Comment:
I was expecting this to replace the existing make_unlive, make_unlive is not being used in forms-admin or forms-runner